### PR TITLE
Adding Rise In bootcamp community repos

### PR DESCRIPTION
**Pull request by Rise-In**

### CODE REVIEW

**Comment by Rise-In** [comment]
Adding Rise In bootcamp community repos

**Comment by Rise-In** [comment]
Adding Rise In bootcamp community repos

**Comment by Rise-In** [comment]
Adding Rise In bootcamp community repos

**Comment by Rise-In** [comment]
Adding Rise In bootcamp community repos

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 1]
```diff
@@ -45419,3 +45413,204 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/berkaykarul/sushiswap"
+
+[[repo]] 
+url = "https://github.com/Jaeson-gram/Solidity-BNB-Chain-Development-Bootcamp-Final-Project.git"
+
+[[repo]] 
+url = "https://github.com/Tejaswini262004/CS50-Final-Project"
+
+[[repo]] 
+url = "https://gist.github.com/arhansuba/86f62231d8d26350734bd929b44e0603"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45553]: not a crypto repo

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 2]
```diff
@@ -45419,3 +45413,204 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/abdulhakim-altunkaya/bnb_patika_AMM"
+
+[[repo]] 
+url = "https://github.com/theritalin/DiceGameDesignPatika"
+
+[[repo]] 
+url = "https://github.com/kaankacar/PTK-Staking-App-For-Patika"
+
+[[repo]] 
+url = "https://github.com/14metehan53/bnb-chain-final-project/tree/master"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45568]: please fix this kind of URLs, format should be "https://github.com/org/repo"

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 3]
```diff
@@ -45419,3 +45413,204 @@ url = "https://github.com/zzzuhaibmohd/DeFiHackLabs"
...
+url = "https://github.com/MeryemSulum/Cargo_Tracking_Dapp/tree/main"
+
+[[repo]] 
+url = "https://github.com/adeuSH1/BNB_bootcamp"
+
+[[repo]] 
+url = "https://github.com/srhtdmrkl/smart-tenders"
+
+[[repo]] 
+url = "https://github.com/Feyzanur25/blokchain/blob/main/smart_contract_frontend/package-lock.json"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:45589]: wrong URL

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 4]
```diff
@@ -18171,9 +18168,6 @@ url = "https://github.com/GulfCoinOfficial/gulf-token-smartContract"
 [[repo]]
 url = "https://github.com/guneyee/BNB-Chain-Safe-Smart-Contract-Frontend"
 
-[[repo]]
-url = "https://github.com/guneyee/Safe-Smart-Contract-on-BNB-Chain"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:18175]: why delete this URL?

File: data/ecosystems/b/bnb-chain-bsc.toml [hunk 5]
```diff
@@ -2273,9 +2273,6 @@ missing = true
 [[repo]]
 url = "https://github.com/AhmedSalikRiaz/NFT_marketplace_using_React_and_Moralis"
 
-[[repo]]
-url = "https://github.com/ahmetenesdur/chat-dapp"
```
Comment (eherrerosj) [inline data/ecosystems/b/bnb-chain-bsc.toml:2277]: why delete this URL?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
good first step, needs some work, left some comments. There quite a few URLs that need the right formatting

File: data/ecosystems/b/bnb-chain-bsc.toml (hunk 5 above)
Comment (Rise-In) [reply data/ecosystems/b/bnb-chain-bsc.toml:2277]: We noticed some duplicate repos and removed them

File: data/ecosystems/b/bnb-chain-bsc.toml (hunk 3 above)
Comment (Rise-In) [reply data/ecosystems/b/bnb-chain-bsc.toml:45589]: Thank you very much for the return. Links will be updating

**Comment by Rise-In** [comment]
Adding Rise In bootcamp community repos and removing duplicate links

**Comment by eherrerosj** [comment]
> Adding Rise In bootcamp community repos and removing duplicate links

Don't forget fixing [this kind of malformed URLs](https://github.com/electric-capital/crypto-ecosystems/pull/771/files#r1434330187)

**Comment by Rise-In** [comment]
Removed empty repo links for BNB Chain and corrected incorrect links

**Comment by Rise-In** [comment]
Our branches are out of date, should we send a new request for these updates?

**Comment by eherrerosj** [comment]
> Our branches are out of date, should we send a new request for these updates?

You could do that or manually solve the conflicts. In any case I'm still seeing incorrect links and repos that shouldn't be deleted. Let me know when you work on those and I'll do another review. Thanks!

**Pull request closed without merging**